app/scripts/controllers/cluster.js

```diff
@@ -157,69 +201,87 @@ angular.module('cluster', ['stats', 'sql', 'common', 'nodeinfo'])
 
     var drawGraph = function(node) {
 
-      $scope.cpuData = [
-        {
-          'key': 'System',
-          'values': [['CPU', node.cpu.system]],
-          'color': COLORS.used
-        },
-        {
-          'key': 'User',
-          'values': [['CPU', node.cpu.user]],
-          'color': '#5d89fe'
-        },
-        {
-          'key': 'Idle',
-          'values': [['CPU', Math.max(0, 100-node.cpu.system-node.cpu.user-node.cpu.stolen)]],
-          'color': COLORS.free
-        },
-        {
-          'key': 'Stolen',
-          'values': [['CPU', node.cpu.stolen]],
-          'color': '#f6bb41'
-        }];
-
-      $scope.heapData = [
-        {
-          'key': 'Used',
-          'values': [['HEAP', node.heap.used]],
-          'color': COLORS.used
-        },
-        {
-          'key': 'Free',
-          'values': [['HEAP', node.heap.max - node.heap.used]],
-          'color': COLORS.free
-        }
-      ];
-
-      $scope.diskUsageData = [
-        {
-          'key': 'Used',
-          'values': [['Disk Usage', node.fs.used]],
-          'color': COLORS.used
-        },
-        {
-          'key': 'Free',
-          'values': [['Disk Usage', node.fs.available]],
-          'color': COLORS.free
-        }
-      ];
-
-      $scope.processCpuData = [
-        {
-          'key': 'Used',
-          'values': [['Process CPU', Math.min(100.0, node.proc_cpu.percent / node.num_cores)]],
-          'color': COLORS.used
-        },
-        {
-          'key': 'Idle',
-          'values': [['Process CPU', Math.max(0, 100.0 - node.proc_cpu.percent / node.num_cores)]],
-          'color': COLORS.free
-        }
-      ];
+      $scope.cpuData = [{
+        'key': 'System',
+        'values': [{
+          "label": "CPU",
```

Comment:
When I run `grunt server` and try and change a file, jshint errors here and below with "Mixed double and single quotes."